examples/v0/account_management/create_customer.py

```diff
@@ -14,7 +14,7 @@
 """This example illustrates how to create a new customer under a given
```

Comment:
*nitpick*: 1-liner summary should be 1 line.

Reply:
I'm not sure I follow. This is formatted similar to comments in other examples, see https://github.com/googleads/google-ads-python/blob/master/examples/v0/account_management/get_account_information.py#L14

Reply:
Summary line appears > 80 chars.

Reply:
Weird, this line isn't over 80 chars - I wonder if it's just silliness with out GitHub is displaying the diff.